pywebostv/controls.py

```diff
@@ -47,12 +47,16 @@ def process_payload(obj, *args, **kwargs):
         return obj
 
 
-def standard_validation(payload):
-    if not payload.pop("returnValue", None):
+def standard_validation(payload, key="returnValue"):
+    if not payload.pop(key, None):
         return False, payload.pop("errorText", "Unknown error.")
```

Comment:
A tiny nit: `return False, payload.pop(*key*, "Unknown error.")`

Reply:
I don't quite understand it... you suggest the following:

```
def standard_validation(payload, key="returnValue"):
    if not payload.pop(key, None):
        return False, payload.pop(key, "Unknown error.")
    return True, None
```
right?

which basically is equivalent to:

```
def standard_validation(payload, key="returnValue"):
    if not payload.pop(key, None):
        return False, "Unknown error."
    return True, None
```

Please explain ;)

Reply:
My bad -- I didn't read the diff correctly. You're right!